plugins/haskell.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "haskell",
+  "version": "0.0.1",
+  "match": "^(ghc|haskell\\.compiler\\.(.*))$",
+  "readme": "Haskell plugin",
+  "packages": [
+    "path:{{ .Virtenv }}"
```

Comment:
I'm struggling to interpret this line. How should I read it in my head?

Reply:
Packages are allowed to be flakes. For example

`path:relative/path/to/my/flake`

where `relative/path/to/my/flake` has `flake.nix` in it.

This plugin creates a `{{ .Virtenv }}/flake.nix`